packages/core/core/src/CommittedAsset.js

```diff
@@ -21,16 +23,19 @@ export default class CommittedAsset {
 
   constructor(value: Asset, options: ParcelOptions) {
     this.value = value;
+    this.key = getFeatureFlag('parcelV3')
+      ? this.value.id
+      : this.value.contentKey;
```

Comment:
This is probably fine for now but I'm not sure if not having a contentKey is going to cause problems anywhere else. I believe id is a hash of the filePath, where as content key might be more specific and include contents as well.

Reply:
I reckon the content key is a cached ID. That means when the inputs change, the id may change but not this content key.

Reply:
It's just to get it working as a first pass, happy for it to change when necessary